sfepy/discrete/fem/mappings.py

```diff
@@ -209,6 +203,8 @@ def get_mapping(self, qp_coors, weights, poly_space=None, ori=None,
             The coordinates of the integration points.
         weights:
             The integration weights.
+        bf: numpy.ndarray
+            The base functions.
```

Comment:
We should use 'basis function' instead of 'base function'. Mea culpa...

Reply:
I agree.